go/client/cmd_pgp_select.go

```diff
@@ -31,15 +31,42 @@ func (v *CmdPGPSelect) ParseArgv(ctx *cli.Context) (err error) {
 	}
 	if err == nil {
 		v.multi = ctx.Bool("multi")
-		v.skipImport = ctx.Bool("no-import")
-		v.onlyImport = ctx.Bool("only-import")
-		if v.onlyImport && v.skipImport {
-			err = fmt.Errorf("Can specify only one of --no-import OR --only-import")
+		v.importSecret = ctx.Bool("import")
+		v.noPublish = ctx.Bool("no-publish")
+		if v.noPublish && !v.importSecret {
+			err = fmt.Errorf("Nothing to do - refused to publish with \"--no-publish\" but did not ask to import secret key (\"--import\")")
 		}
 	}
 	return err
 }
 
+const selectDisclaimer = `You are selecting a PGP key from your local GnuPG keychain, and
+will publish a statement signed with this key to make it part of 
+your Keybase.io identity.
+
+You will be prompted by GnuPG to unlock your private key - it will
+be used to make a signature that you are the owner of said key.
+
+You can also import the secret key to *local*, *encrypted* Keybase
+keyring, to be able to decrypt or sign using Keybase client. To do
```

Comment:
`to be able to decrypt or sign using Keybase Client` -> `enabling decryption and signing with the Keybase client`.